lms/djangoapps/grades/course_data.py

```diff
@@ -100,13 +100,13 @@ def __unicode__(self):
         return u'Course: course_key: {}'.format(self.course_key)
 
     def full_string(self):
-        if self._effective_structure:
+        if self.effective_structure:
             return u'Course: course_key: {}, version: {}, edited_on: {}, grading_policy: {}'.format(
                 self.course_key, self.version, self.edited_on, self.grading_policy_hash,
             )
         else:
             return u'Course: course_key: {}, empty course structure'.format(self.course_key)
 
     @property
-    def _effective_structure(self):
+    def effective_structure(self):
```

Comment:
**Note to reviewers:** Made this property public so certain callers can use whichever structure is available, including the user-agnostic collected-block-structure.